### DISCUSSION TITLE
There is a way to export the model generated by your addon?

### DISCUSSION THREAD

**Discussion by ericdotjs** [BradyAJohnston/MolecularNodes#140] [Q&A]
# There is a way to export the model generated by your addon?

Hello Braddy, I hope you're fine.

I want to ask you about if this addon has the export function. I have read you documentation and watched some videos from your Youtube Channel and all your content had helped to me. However, I'd like to work with this models in something Educational but I need export his models in .FBX format for example.

As I see, this addon implements a new modifier and when I export a model from a .pdb this modifier it's applied then the model doesn't look like before the moddifier was applied.

And I got lost, I don't know if I missing something about your documentation or even i don't know  sure if this addon just was desing for make simulations, visualizations but not export and use these models.

I send you two picture that may be can ilustrate my point I hope you can help me. Thank you

# Just the model generated by your addon.
![img1](https://imgur.com/FrzIcPX.png)

# When the model is exported 
![img2](https://imgur.com/TRtCT2G.png)

**Comment by ericdotjs** [comment]
# Update 

Sorry I found the way, I just had to do some settings with the geometry nodes, but thank so much for doing this addon.

**Reply by BradyAJohnston** [reply]
I haven't played around much with exporting from Blender. Point clouds aren't supported, but the rest of the meshes should work if you apply the modifier. Can you share what settings you had to change to get it to work?

**Comment by ericdotjs** [comment]
Sure. I just add this node into the geometry nodes called "release-instance" that converts the model in a mesh, then you just apply the modifier. If you wan
t to be sure of that, after the last step just press right click and press in convert to mesh and that's all.

For the texture you can make a texture of a way procedural using the bake system with Cycles.

# Adding the  node
![image](https://user-images.githubusercontent.com/82389602/213615105-94295423-e8bc-4574-af3c-f8ffd66a9447.png)

# Aftter adding the node "release instance"

![image](https://user-images.githubusercontent.com/82389602/213615579-85622a04-c5f0-4a31-a6c8-ef5536fc9038.png)

and then you will be able to export from another 3d Software.

**Reply by BradyAJohnston** [reply]
ah yes the instances need to be realised before they'll be turned into geometry, interesting that they aren't realised when you apply the modifier though by default. Thanks for the followup!

**Reply by ericdotjs** [reply]
Yes, I didn't get at the beginning but I forgot that step in the geometry nodes... what a shame but well, it works!
